packages/react/src/menu/positioner/MenuPositioner.tsx

```diff
@@ -80,6 +81,7 @@ export const MenuPositioner = React.forwardRef(function MenuPositioner(
   if (parent.type === 'menu') {
     computedSide = computedSide ?? 'inline-end';
     computedAlign = computedAlign ?? 'start';
+    collisionAvoidance = POPUP_COLLISION_AVOIDANCE;
```

Comment:
It might be confusing if we silently overwrite the prop.

```suggestion
    collisionAvoidance = componentProps.collisionAvoidance ?? POPUP_COLLISION_AVOIDANCE;
```